packages/teams-js/src/internal/utils.ts

```diff
@@ -280,8 +280,11 @@ export function createTeamsAppLink(params: pages.NavigateToAppParams): string {
   if (params.webUrl) {
     url.searchParams.append('webUrl', params.webUrl);
   }
-  if (params.channelId || params.subPageId) {
-    url.searchParams.append('context', JSON.stringify({ channelId: params.channelId, subEntityId: params.subPageId }));
+  if (params.chatId || params.channelId || params.subPageId) {
+    url.searchParams.append(
+      'context',
+      JSON.stringify({ chatId: params.chatId, channelId: params.channelId, subEntityId: params.subPageId }),
```

Comment:
If not available, do these props get stringified with a undefined value or dropped? Ideally, we should only add these to the URL if they are define.

Reply:
if the value is undefined it would get dropped

![Screenshot 2024-07-10 at 6 25 56 PM](https://github.com/OfficeDev/microsoft-teams-library-js/assets/153236114/522cc6c8-bb5b-4fc1-9be8-8587d66519ef)